packages/sprotty/src/features/routing/routing.ts

```diff
@@ -147,7 +148,7 @@ function isMultipleEdgesRouter(
 
 /** A postprocessor that is applied to all routes, once they are computed. */
 export interface IEdgeRoutePostprocessor {
-    apply(routing: EdgeRouting): void;
+    apply(routing: EdgeRouting, parent?: SParentElementImpl): void;
```

Comment:
I think the new argument can be mandatory. The change would be breaking only for users who are calling this service in their own code, but that's very unlikely.